src/features/governance/hooks/useVote.ts

```diff
@@ -19,10 +19,9 @@ export const useVote = () => {
   const { address } = useAccount();
   const chainId = useChainId();
   const network = chainIdToChain(chainId);
-
   const getVoteCacheKey = useCallback(
-    (id: string) => {
-      return `${network.name}:voteProposal:${id}`;
+    (id: string, addr: string) => {
+      return `${network.name}:${addr}:voteProposal:${id}`;
```

Comment:
!important! when caching must be by the address, sadly this WILL break the cache for old votes there is no way around that ce la vi

Reply:
c'est la vie indeed